openage/game/main.py

```diff
@@ -36,6 +36,19 @@ def init_subparser(cli: ArgumentParser) -> None:
         help="Check if the assets are up to date"
     )
 
+    cli.add_argument(
+        "--window-size", nargs=2, type=int, default=[1024, 768],
+        metavar=('WIDTH', 'HEIGHT'),
+        help="initial window size in pixels, e.g., --window-size 1024 768")
```

Comment:
```suggestion
        help="Initial window size in pixels")
```

You don't need the example because Python's argparse will show enough detail.